src/aiida_quantumespresso/parsers/base.py

```diff
@@ -25,39 +88,127 @@ def emit_logs(self, logging_dictionaries, ignore=None):
                 'error': ['Self-consistency was not achieved']
             }
 
-        :param logging_dictionaries: log dictionaries
+        :param logs: log dictionaries
         :param ignore: list of log messages to ignore
         """
         ignore = ignore or []
 
-        if not isinstance(logging_dictionaries, (list, tuple)):
```

Comment:
I was also inclined to remove this, but then noticed how the `pw.x` parser relies on this heavily and I'm trying not to touch this one too much to make your rebase of the XML work easier. It did raise a question: should we:

1. Instantiate a _single_ log container, that is passed _through_ the parsing methods, as I'm doing in this PR?
2. Instantiate _multiple_ log containers and return them as a list, as is done e.g. in the `PwParser`?

I'm marginally in favor of [1], but [2] would also work, and allow us to keep the logs separated, which might have benefits?